tools/sqf_validator.py

```diff
@@ -117,7 +118,7 @@ def popClosing():
                             if (c not in [' ', '\t', '\n', '/']): # keep reading until no white space or comments
                                 checkForSemiColumn = False
                                 if (c not in [']', ')', '}', ';', ',', '&', '!', '|', '='] and not validKeyWordAfterCode(content, indexOfCharacter)): # , 'f', 'd', 'c', 'e', 'a', 'n', 'i']):
-                                    print("ERROR: Possible missing semi-column ';' detected at {0} Line number: {1}".format(filepath,lineNumber))
+                                    print("ERROR: Possible missing semi-colon ';' detected at {0} Line number: {1}".format(filepath,lineNumber))
```

Comment:
https://en.wikipedia.org/wiki/Semicolon
`The semicolon or semi colon` either no `-` or whitespace.